src/components/Web3Status/index.tsx

```diff
@@ -230,6 +231,7 @@ function Web3StatusInner() {
   } else if (account) {
     return (
       <Web3StatusConnected data-testid="web3-status-connected" onClick={toggleWallet} pending={hasPendingTransactions}>
+        {isNavbarEnabled && !hasPendingTransactions && <StatusIcon connectionType={connectionType} />}
```

Comment:
Restructure this around the experiment, so that you only do the check for `isNavBarEnabled` once. The other checks can be duplicated.

    {isNavBarEnabled ?
      (
        {hasPendingTransactions ? (...) : (...)}
        {walletIsOpen ? <ChevronUp /> : <ChevronDown />}
      ) : (
        hasPendingTransactions ? (...) : (...)
      )
    }

This will make the code mirror the codepaths better, and will make it easier to remove the old codepath (`!isNavBarEnabled`) with minimal code changes.